src/components/MenuComponent.jsx

```diff
@@ -0,0 +1,318 @@
+            <Link to='/historycomponent'>
+               <i className='fa fa-history'></i>
+            </Link>
+          </li>
+        </ul>
+      </Menu>
+      <Switch>
+        <Route path='/karaktercomponent'>
+          <Karaktercomponent />;
+        </Route>
+
+        <Route path='/knowledgecomponent'>
+          <Knowledgecomponent />;
+        </Route>
+        <Route path='/achievementscomponent'>
+          <Achievementscomponent />
+        </Route>
+        <Route path='/questscomponent'>
+          <Questscomponent />
+        </Route>
+        <Route path='/historycomponent'>
+          <Historycomponent />
+        </Route>
+      </Switch>
+    </Router>
+  );
+}
+
+function Karaktercomponent() {
+  return (
+    <KarakterWrapper>
+      <KarakterContent>
+        <ImgWrapper>
+          <img src={karakter_img} alt='foto_kim' />
+        </ImgWrapper>
+
+        <ContentText>
+          <p>
+            <span>Kim Schenk</span>
+            <span>Frontend Developer</span>
+            <span>FIOD</span>
+          </p>
+        </ContentText>
+      </KarakterContent>
+
+      <KarakterTable>
+        <Table>
+          <div className='table__row'>
+            <div className='row__items'>
+              <div>skills</div>
+              <div>kim</div>
+              <div>html</div>
+              <div>css</div>
+              <div>javascript</div>
+              <div>typeschript</div>
+            </div>
+          </div>
+
+          <div className='table__column'>
+            <div className='column__items'>
+              <div>0%</div>
+              <div>25%</div>
+              <div>50%</div>
+              <div>75%</div>
+              <div>100%</div>
+              <div>125%</div>
+              <div>150%</div>
+              <div>175%</div>
+              <div>200%</div>
+
+              <div className='bar1'>200%</div>
+              <div className='bar2'>25%</div>
+              <div className='bar3'>100%</div>
+            </div>
+          </div>
+        </Table>
+      </KarakterTable>
+    </KarakterWrapper>
+  );
+}
+
+function Knowledgecomponent() {
+  return <h2>KnowledgeComponent</h2>;
+}
+function Achievementscomponent() {
+  return <h2>AchievementsComponent</h2>;
+}
+function Questscomponent() {
+  return <h2>QuestsComponent</h2>;
+}
+function Historycomponent() {
+  return <h2>HistoryComponent</h2>;
+}
+
+const Menu = styled.nav`
+  height: 90px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  ul {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    background: #f97970;
+    padding: 15px 0;
+    place-items: center center;
+  }
+
+  li {
+    list-style-type: none;
+  }
+
+  li:hover::after {
+    position: absolute;
+    top: -125px;
+    left: 7%;
+    padding: 10px;
+    border-radius: 50px;
+    font: oblique small-caps bold 30px/21px Georgia, serif;
+    background: #fff;
+  }
+  li:nth-child(2):hover::after {
+    content: 'Karakter';
+    top: -75px;
```

Comment:
wat ik al eerder aangaf is het handiger en makkelijker om deze getallen niet te hardcoden maar dit relatief aan elk parent element te doen. Kun je dat invoeren?